app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java

```diff
@@ -49,8 +78,49 @@ protected void onCreate(Bundle savedInstanceState) {
         RecyclerView mRecView = this.findViewById(R.id.recycler_view_maps);
         mController.mRecView = mRecView;
         mController.setDynamicRecyclerView(this);
-        mSearchView = this.findViewById(R.id.tb_toolbarsearch);
-        mController.mSearchView = mSearchView;
+
+        setUpSearch();
+    }
+
+    private void setUpSearch(){
+        mSearchView = (FloatingSearchView) findViewById(R.id.search_view);
+        mSearchView.setOnQueryChangeListener(new FloatingSearchView.OnQueryChangeListener() {
+            @Override
```

Comment:
probably should have newline before this? not sure, resolve if you think this is fine